project.clj

```diff
@@ -8,32 +12,77 @@
 
   :pedantic? :abort
 
-  :dependencies [[org.jruby/jruby-core "1.7.26"
-                  :exclusions  [com.github.jnr/jffi
-                                com.github.jnr/jnr-x86asm
-                                joda-time
-                                org.yaml/snakeyaml]]
+  ;; Note that there is a fair amount of duplication between the
```

Comment:
@rlinehan I reworked the dependencies for reuse in 6fa3b1d.  I think this will be an improvement overall.  I'd be curious to hear what you think.